cudamapper/src/overlapper.cpp

```diff
@@ -203,5 +204,39 @@ void Overlapper::post_process_overlaps(std::vector<Overlap>& overlaps)
         overlaps.push_back(fused_overlap);
     }
 }
+
+void Overlapper::rescue_overlap_ends(std::vector<Overlap>& overlaps,
+                                     const io::FastaParser& query_parser,
+                                     const io::FastaParser& target_parser,
+                                     int max_extension,
```

Comment:
As discussed before: use types with specified number of bits, use `const` for pass by value parameters in function definitions where appropriate

Reply:
Addressed by 8e951f9a.